static/js/containers/TermsOfServicePage.js

```diff
@@ -31,19 +31,106 @@ class TermsOfServicePage extends React.Component {
   }
 
   render() {
+    let linkToEdxTermsOfService = (
+      <a href="https://www.edx.org/edx-terms-service" target='_blank'>https://www.edx.org/edx-terms-service</a>
+    );
     return <div>
       <h3>
         MIT Micromasters Terms of Service
       </h3>
 
       <p>
-        Please read through the terms of service. To enroll in MIT Micromasters you
-        must agree to the terms of service.
+        Welcome to the MITx MicroMaster's website (the “Site”).  By accessing this Site, users agree to be
```

Comment:
A question and a comment:

1. Do we need to escape the single quote? GitHub's syntax checker seems upset. 
2. Let's use straight quotes, i.e. 

```
(the "Site").
```